kamrons-game.py

```diff
@@ -9,6 +9,14 @@ def typewriter(str):
         sleep(0.07)
     print("")
 
+
```

Comment:
It would be nice to document what this method does so you don't forget later.

```
def typewriter2(str):
    """
    Output the given string. Slowly.
    :param str: The string to output. Slowly.
    """
    for letter in str:
        sys.stdout.write(letter)
        ........
```

Reply:
done